.config/playwright-setup.js

```diff
@@ -115,7 +115,12 @@ ${demos.map((d) => `from demo.${d}.run import demo as ${d}`).join("\n")}
 
 app = FastAPI()
 ${demos
-	.map((d) => `app = gr.mount_gradio_app(app, ${d}, path="/${d}")`)
+	.map(
+		(d) =>
+			`app = gr.mount_gradio_app(app, ${d}, path="/${d}", max_file_size=${
+				d == "upload_file_limit_test" ? 1 : "None"
```

Comment:
Might be worth adding a note in `demo/upload_file_limit_test` that the upload limit is set in `.config/playwright-setup.js` because
(1) users might be confused since no limit is explicitly set in that demo
(2) if we were to change the test later, we might not remember where the limit is actually being set

Reply:
Done!